local/network.go

```diff
@@ -166,15 +167,15 @@ func (npc *nodeProcessCreator) NewNodeProcess(config node.Config, args ...string
 	if config.RedirectStdout {
 		stdout, err := cmd.StdoutPipe()
 		if err != nil {
-			return nil, fmt.Errorf("Could not create stdout pipe: %s", err)
+			return nil, fmt.Errorf("couldn't create stdout pipe: %s", err)
```

Comment:
not arguing if `couldn't` is better than `could not` :smile:

Reply:
lol I mainly changed this because my linter was complaining about an error message starting with a capital letter